wagtail/embeds/oembed_providers.py

```diff
@@ -770,6 +763,140 @@
     wistia,
     wordpress,
     yandex,
-    yfrog,
     youtube,
 ]
+
+
+'''
+updated_urls_provider=[
```

Comment:
Thanks @dushyantbhati2 - these notes can just go on the PR as a comment or description update. No need to put this in the code.